src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2102,25 +2141,25 @@
           }
         },
 
-        "notificationsMetadata": {
+        "notificationDescriptors": {
           "description": "An array of message descriptor objects relevant to the notifications related to the configuration and runtime execution of the component.",
```

Comment:
>message descripto [](start = 38, length = 17)

"message descriptor" =>"output descriptor" (assuming we keep the name `outputDescriptor`, which I'm struggling with).

Reply:
Here and on `ruleDescriptors.description` below.

---
In reply to: [256190665](https://github.com/Microsoft/sarif-sdk/pull/1280#discussion_r256190665) [](ancestors = 256190665)